src/path.rs

```diff
@@ -21,6 +26,12 @@ pub trait PathExt: crate::Sealed {
 }
 
 impl PathExt for Path {
+    #[cfg(feature = "path_try_exists")]
+    fn fs_err_try_exists(&self) -> io::Result<bool> {
+        self.try_exists()
+            .map_err(|source| Error::build(source, ErrorKind::FileExists, self))
```

Comment:
Weirdly enough, `try_exists` on Path is stabilized by `fs::try_exists` is not. I couldn't follow the same pattern used by the rest of the associated trait functions.